lua/telescope/builtin/lsp.lua

```diff
@@ -55,11 +55,12 @@ lsp.document_symbols = function(opts)
     return
   end
 
+  opts.ignore_filename = opts.ignore_filename or true
```

Comment:
We can use `utils.get_default()` here as well. Someone made a fix for it so it also works with `:Telescope fn opts=false`. Because currently we are getting a string and that breaks things.